src/content/docs/properties/about-properties/index.mdx

```diff
@@ -27,29 +27,37 @@ You can choose to make a property ‘public’ so it can be made available to be
 
 To get you started, we’ve included a set of common properties. You can easily identify these as the code key starts with `kp-` meaning Kinde-provided. You cannot edit or delete these properties.
 
-## C**ustom properties**
+## Custom properties
 
-The needs of every business is different, that’s why you can create your own properties and use them any way you want. For example, you might want to create a property to collect marketing information or customer relationship data.
+The needs of every business is different, that’s why you can create your own properties and use them any way you want. For example, you might want to create a property to collect marketing information, identifiers from your app, or customer relationship data.
 
 ## Property categories
 
-Properties can be organized using [property categories](/properties/work-with-properties/property-groups/). When you create a category, it becomes a heading in a user or organization record.
+Properties can be organized using [property categories](/properties/work-with-properties/property-groups/). When you create a category, it becomes a heading in a user, application, or organization record. Property categories ensure similar fields are grouped together for easier display. 
 
-Categories ensure similar fields are grouped together for easier display. For example, you might have a group for ‘billing information’ and a separate group for ‘delivery information’, even though they contain similar properties.
+### Organization properties example
 
 | Category             | Properties                                         | For          |
 | -------------------- | -------------------------------------------------- | ------------ |
 | Billing information  | Billing contact, Billing address, Billing phone    | Organization |
 | Delivery information | Delivery contact, Delivery address, Delivery phone | Organization |
 
-And example of user categories might be:
+### User properties example
 
 | Category         | Properties         | For  |
 | ---------------- | ------------------ | ---- |
 | Accounts contact | Name, email, phone | User |
 | Next of kin      | Name, email, phone | User |
 
-## **How properties are displayed in Kinde**
+### Application properties example
+
+| Category      | Properties         | For         |
+| -----------   | ------------       | ----        |
+| Internal ID   | Internal app ID    | Application |
+| Internal Name | Internal app name  | Application |
+
+
```

Comment:
@clairekinde11 do we mention that another common use case is for correlating M2M applications with organizations?